### ISSUE TITLE
 Ambient modules cannot be nested in other modules or namespaces.

### ISSUE THREAD

**Issue by pflannery** [Microsoft/vscode-extension-vscode#15]
I'm seeing

`node_modules/vscode/vscode.d.ts(2761,16): error TS2435: Ambient modules cannot be nested in other modules or namespaces.`

when running 

`node ./node_modules/vscode/bin/compile`

this causes issues for things like `vsce package` and project references to `vscode`

Is there something I'm doing wrong or is this a bug?

**Comment by pflannery** [comment]
with v0.11.1 I now see

```
> node ./node_modules/vscode/bin/compile

node_modules/vscode/vscode.d.ts(2762,2): error TS2666: Exports and export assignments are not permitted in module augmentations.
```

These kind of errors prevent us from testing extensions on medium's like travis. (Locally we can amend the file and the compile will work)
Would be nice to get this stable for extension testing.

**Comment by bpasero** [comment]
@pflannery which typescript version do you have installed? I wonder if the wrong typescript is being picked up?

**Comment by pflannery** [comment]
@bpasero The issue onlyoccurs on travis and I have 1.8.2 defined as my devDependencies. 
I have windows 10 and don't get any problems running my tests.

travis fail link: https://travis-ci.org/pflannery/vscode-versionlens/builds/112650150#L183

**Comment by bpasero** [comment]
@dbaeumer any ideas?

**Comment by bpasero** [comment]
Found it, fixed with vscode@0.11.3

**Comment by pflannery** [comment]
awesome. thanks
